src/NFTWorker.ts

```
@@ -201,16 +192,13 @@ export default class NFTWorker {
                         this.found(null);
                         break;
                     }
-                    case "error": {
-                        console.log("NFTWorker : error");
-                        var event = new Event("nftError");
-                        this.target.dispatchEvent(event);
-                        break;
-                    }
                 }
                 this._processing = false;
                 trackUpdate();
             };
+            this.worker.onerror = (err: any) => {
+                console.error("Worker error from NFTWorker: ", err);
```

if its a library probably should throw an error to the upper component!
I'll create an issue, i'd like to research more on how-to do it

---

`this.worker.onerror` actually does nothing more or less, but i'd like to improve in a future the error handling. As i understood to listen errors from the worker we should set up another some `throw Error` inside the Worker.
ARnft in general should have a better handling errors system i think.